File: Programming-languages-and-laboratory/Python/Exercises/26_compito_java_20181119/Studente.py
from Partecipante import Partecipante


class Studente(Partecipante):
    def __init__(self, codice, tipo, nome, cognome, corso, universita, anno_corso):
        super().__init__(codice, tipo, nome, cognome)
        self._corso = corso
        self._universita = universita
        self._anno_corso = anno_corso

    def get_universita(self):
        return self._universita

    def __str__(self):
        return super().__str__() + "\t" + self._corso + "\t" + self._universita + "\t" + str(self._anno_corso) + "\t-\t-\t-"

File: II - Linguaggi di programmazione e laboratorio/Esame/Java/26_compito_java_2019_11_19/Studente.java
import java.util.*;

public class Studente extends Partecipante {

	private String corso, universita;
	private int anno;

	public Studente(int codice, String nome, String cognome, String corso, String universita, int anno) {
		super(codice, nome, cognome, "studente");
		this.corso = corso;
		this.universita = universita;
		this.anno = anno;
	}

	public String getUni() {
		return universita;
	}

	public String toString() {
		return super.toString() + corso + "\t" + universita + "\t" + anno + "-\t-\t-";
	}

}